package.json

```diff
@@ -16,14 +16,14 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.19.1",
-    "@bcrs-shared-components/breadcrumb": "2.1.0",
-    "@bcrs-shared-components/confirm-dialog": "1.2.0",
-    "@bcrs-shared-components/corp-type-module": "1.0.7",
-    "@bcrs-shared-components/court-order-poa": "2.1.0",
-    "@bcrs-shared-components/enums": "1.0.22",
-    "@bcrs-shared-components/interfaces": "1.0.42",
-    "@bcrs-shared-components/staff-comments": "1.3.0",
-    "@bcrs-shared-components/staff-payment": "2.1.0",
+    "@bcrs-shared-components/breadcrumb": "2.1.1",
+    "@bcrs-shared-components/confirm-dialog": "1.2.1",
+    "@bcrs-shared-components/corp-type-module": "1.0.8",
+    "@bcrs-shared-components/court-order-poa": "2.1.1",
+    "@bcrs-shared-components/enums": "1.0.23",
+    "@bcrs-shared-components/interfaces": "1.0.44",
+    "@bcrs-shared-components/staff-comments": "1.3.1",
+    "@bcrs-shared-components/staff-payment": "2.1.1",
```

Comment:
This was due to a re-version of all components due to tags not fetched (no code change).

Eg, Breadcrumb went from 2.0.7 -> 2.1.0 -> 2.0.8 -> 2.0.9 -> 2.1.1 🤦‍♂️